templates/e3sm_diags.bash

```diff
@@ -50,6 +52,19 @@ cd climo
 cp -s ${climoDir}/${case}_*_${Y1}??_${Y2}??_climo.nc .
 cd ..
 
+# Create local links to input ts files
+# TODO: tsDir doesn't exist. How did `clim` above ge constructed?
+tsDir={{ output }}/post/atm/{{ grid }}/ts/{{ '%dyr' % (ts_year2-ts_year1+1) }}
+mkdir -p ts_links
+cd ts_links
+# https://stackoverflow.com/questions/27702452/loop-through-a-comma-separated-shell-variable
+for v in $(echo {{ vars }} | sed "s/,/ /g")
+do
+    xml_name=${v}_${Y1}01_${Y2}12.xml
+    cdscan -x ${xml_name} ${tsDir}/${v}_*.nc
```

Comment:
This will actually create a `xml` file for all years of a variable (e.g., 1-10, 11-20, ... 91-100) rather than just the years the diags cover (e.g., 1-20). 

It may be a challenge to determine what the minimum list of files to run `cdscan` on would be.